contrib/docker/Dockerfile.root

```diff
@@ -1,5 +1,9 @@
 FROM dealii/dealii:master-focal
 
-LABEL maintainer="luca.heltai@gmail.com"
-
 USER root
+
+ENV HOME /root
+
+WORKDIR /root
+
+ENV OMPI_ALLOW_RUN_AS_ROOT 1
```

Comment:
After a quick search, I've spotted some references to `OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1`. Do we perhaps need this as well?

Reply:
Yes, I believe that the conclusion of the discussion we had with the OpenMPI folks was that you had to set both environment variables.

Reply:
Indeed. Added.